packages/react/src/patterns/blocks/ContentBlockSimple/ContentBlockSimple.js

```diff
@@ -0,0 +1,94 @@
+/**
+ * Copyright IBM Corp. 2016, 2018
+ *
+ * This source code is licensed under the Apache-2.0 license found in the
+ * LICENSE file in the root directory of this source tree.
+ */
+
+import { ContentBlock } from '../../sub-patterns/ContentBlock';
+import { ContentItem } from '../../sub-patterns/ContentItem';
+import { settings as ddsSettings } from '@carbon/ibmdotcom-utilities';
+import { Image } from '../../../components/Image';
+import PropTypes from 'prop-types';
+import React from 'react';
+import { settings } from 'carbon-components';
+
+const { stablePrefix } = ddsSettings;
+const { prefix } = settings;
+
+/**
+ * Content Block - Simple pattern
+ *
+ * @param {object} props props object
+ * @param {string} props.copy Content block simple short copy to support the heading
+ * @param {string} props.linkType link type ( simple | jump | card )
+ * @param {object} props.link link object which includes url, link text and target properties.
+ * @param {string} props.heading Content block simple heading
+ * @param {object} props.cta cta object
+ * @param {object} props.contentGroup chilren content element
+ * @param {string} props.mediaType Media type, video or image
+ * @param {object} props.mediaData Data for renderimg the media
+ * @param {Array} props.items Array of objects with data for ContentItems
+ * @returns {*} Content block simple pattern
+ */
+const ContentBlockSimple = ({
+  copy,
+  heading,
+  items,
+  mediaType,
+  mediaData,
+  cta,
+}) => (
+  <ContentBlock
+    heading={heading}
+    copy={copy}
+    items={items}
+    cta={cta}
+    style
+    data-autoid={`${stablePrefix}--content-block-simple`}
+    customClassName={`${prefix}--content-block-simple ${prefix}--col-lg-8`}>
+    <div
```

Comment:
Is this `div` wrapper necessary`?

Reply:
Hm, yes, to give it a `data-autoid`.

Reply:
Please correct me if I'm becuase I'm still learning on this. Aren't they supposed to be added to the highest container, and interactive/dynamic elements? I neither here.

Reply:
Related to your comment below about `data-autoid` being passed down to `ContentBlock`, which it isn't. This has been fixed.